libs/extensions/angular/image-banner/src/image-banner.component.scss

```diff
@@ -5,6 +5,25 @@ $blur-radius: 60px;
 $max-content-width: 324px;
 $main-image-height-small: 132px;
 $main-image-height-large: 164px;
+$cq-width: 515px;
```

Comment:
Is this a semi-arbitrary value, or is it calculated based on something?

Reply:
The value was read from the width of `ImageBannerComponent` in the default story where the media query small   triggered.

Reply:
I think this value should be 600px 🤔 
When I inspect the original implementation in the [default example](https://extensions.kirby.design/?path=/story/components-banner-image-banner--default), it changes when the iframe is 632px wide. And this setup mimics spacing in `kirby-page` with 16px padding left and right, so the container around the banner is 600px when the layout change happens.

Reply:
done :)